insights/tests/core/spec_factory/test_file_provider.py

```diff
@@ -0,0 +1,28 @@
+import os
+import os.path
+import pytest
+
+from insights.core.spec_factory import FileProvider
+
+
+@pytest.fixture(scope="module")
```

Comment:
I am expecting that we will add more tests to codify the FileProvider API in the future. The same fixture can be shared by all tests for basic properties and features.

Reply:
P.S. I've got another MR ready that adds more tests already. I need this one merged first.